src/common/redis_module/ray_redis_module.cc

```diff
@@ -388,6 +388,46 @@ bool PublishObjectNotification(RedisModuleCtx *ctx,
   return true;
 }
 
+int TableAdd_RedisCommand(RedisModuleCtx *ctx,
+                          RedisModuleString **argv,
+                          int argc) {
+  if (argc != 3) {
+    return RedisModule_WrongArity(ctx);
+  }
+
+  RedisModuleString *id = argv[1];
+  RedisModuleString *data = argv[2];
+
+  // Set the keys in the table
+  RedisModuleKey *key;
+  key = OpenPrefixedKey(ctx, "T:", id, REDISMODULE_READ | REDISMODULE_WRITE);
+  RedisModule_StringSet(key, data);
+  RedisModule_CloseKey(key);
+
+  return RedisModule_ReplyWithSimpleString(ctx, "OK");
+}
+
+int TableLookup_RedisCommand(RedisModuleCtx *ctx,
+                             RedisModuleString **argv,
+                             int argc) {
+  if (argc != 2) {
+    return RedisModule_WrongArity(ctx);
+  }
+
+  RedisModuleString *id = argv[1];
+
+  RedisModuleKey *key;
+  key = OpenPrefixedKey(ctx, "T:", id, REDISMODULE_READ);
+  size_t len;
+  char *buf = RedisModule_StringDMA(key, &len, REDISMODULE_READ);
```

Comment:
Add const, and put asterisk right after the type? `const char* buf`

Reply:
added the const; the asterisk is currently encoded that way in the clang-format file and enforced (we started as a C project, remember), we will change everywhere consistently with a future commit...